controllers/notes.js

```diff
@@ -0,0 +1,34 @@
+const  Note = require('../models/note')
+const {StatusCodes} = require('http-status-codes')
+const {BadRequestError, NotFoundError} = require('../errors')
+
+const getAllNotes = async(req, res) => {
+    res.send('get all notes')
+}
+
+const getNote = async(req, res) => {
+    res.send('get one note')
+}
+
+const createNote = async(req, res) => {
+    //res.send('create a job')
+    req.body.createdBy = req.user.userId
+    const note = await Note.create(req.body)
+    res.status(StatusCodes.CREATED).json({job})
```

Comment:
There is an error here. You should return the note var not the job.

Reply:
I have fixed it